odata-protocol/11.4 Data Modification.md

```diff
@@ -545,7 +545,43 @@ nested delta representation to:
   ]
 }
 ```
+:::
+
+::: example
+Example ##ex: When updating an entity with a 4.01 `PUT` request, the target of a
+non-containment navigation property can be replaced without changing either the
+old or the new targeted entity itself. The targeted entity can then only be
+specified by an entity reference (see [OData-JSON, section "Entity Reference"]($$$OData-JSON$$$#EntityReference)):
+```json
+PUT http://host/service/Employees(7)
+Content-Type: application/json
 
+{
+  "FirstName": "Suzanne",
+  "LastName": "Brown-Johnson",
+  "Manager": {
+    "@context": "$metadata#$ref",
+    "@id": "Employees(6)"
+  }
+}
+```
+
+If the targeted entity in the payload contains some structural properties,
+`PUT` resets all its other structural properties. The following therefore does
+not have the desired effect:
+```json
+PUT http://host/service/Employees(7)
+Content-Type: application/json
+
+{
+  "FirstName": "Suzanne",
+  "LastName": "Brown-Johnson",
+  "Manager": {
+    "@id": "Employees(6)",
+    "EmployeeID": 6
+  }
+}
```

Comment:
Or even stricter?
```suggestion
If the targeted entity in the payload does not have the `@context` of an entity reference,
`PUT` resets all its structural properties not contained in the payload. The following therefore does
not have the desired effect:
```json
PUT http://host/service/Employees(7)
Content-Type: application/json

{
  "FirstName": "Suzanne",
  "LastName": "Brown-Johnson",
  "Manager": {
    "@id": "Employees(6)"
  }
}
```

Reply:
That's too strict: an object only containing `@id` essentially is an entity reference, even without the `@context`.

Reply:
I'd rather remove the `@id` from the "undesired effect" example